easybuild/easyconfigs/g/GEOS/GEOS-3.7.2-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I don't think there is any MPI or BLAS stuff in this package. Perhaps we could move it down to GCC level?